### PR TITLE
Load Mixtral GGUF Model

### PR DESCRIPTION
**Pull request by Uxito-Ada**
## Description

load gguf-format q40 mixtral model into bigdl llm context

### 1. Why the change?

as above

### 2. User API changes

support mixtral for load_gguf

### 3. Summary of the change 

Load Mixtral GGUF Model

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

transformers==4.36.0 for mixtral

### CODE REVIEW

**Comment by jason-dai** [comment]
Need to update CPU/GPU examples

**Comment by jason-dai** [comment]
Please update CPU and GPU example.

**Review by glorysdj — APPROVED** [review]
LGTM

**Comment by Uxito-Ada** [comment]
> Please update CPU and GPU example.

ok, WIP

File: python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Loading GGUF models
-In this directory, you will find examples on how to load GGUF model into `bigdl-llm`. For illustration purposes, we utilize the [llama-2-7b-chat.Q4_0.gguf](https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGUF/tree/main) as a reference LLaMA2 GGUF model.
->Note: Only LLaMA2 family models are currently supported
+In this directory, you will find examples on how to load GGUF model into `bigdl-llm`.
+>Note: Only LLaMA2 family models are currently supported.
```
Comment (jason-dai) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/GGUF/README.md]: Remove `Note: Only LLaMA2 family models are currently supported.`

**Pull request merged**